.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -0,0 +1,95 @@
+name: Vulnerability Scanning on Repo
```

Comment:
# RE: Name

I much prefer seeing file name in job UI

```suggestion
# Vulnerability Scanning on Repo
```


# RE filename

Maybe a shorter filename?
<details>
  <summary>ChatGPT suggestions</summary>
.github/workflows/sec-release.yml
.github/workflows/scan-release.yml
.github/workflows/vuln-check.yml
.github/workflows/comp-scan.yml
.github/workflows/release-audit.yml
.github/workflows/scan-attach.yml
.github/workflows/safe-release.yml
.github/workflows/release-sec.yml
.github/workflows/audit-release.yml
.github/workflows/sec-scan.yml
</details>

Reply:
Feel free to resolve them once read.